.eleventy.js

```diff
@@ -97,15 +97,15 @@ const filenameToData = (f) => ({
     .map(({ name, id }) => ({
       name,
       id,
-      safeScore: getSafeScore(filterByTerritorioId(all, id)),
+      safeScore: Number(getSafeScore(filterByTerritorioId(all, id))),
       subTerritories: territoriesLevel2
         .map(filenameToData)
         .filter(({ file }) => id === file.comunidad)
         .map(({ file, id }) => ({ id, ...file }))
         .map(({ name, id }) => ({
           name,
           id,
-          safeScore: getSafeScore(filterByTerritorioId(all, id)),
+          safeScore: Number(getSafeScore(filterByTerritorioId(all, id))),
```

Comment:
Aquí igual, creo que basta con el cambio a `Number` que has hecho en el valor de retorno de la propia función

```suggestion
          safeScore: getSafeScore(filterByTerritorioId(all, id)),
```